.github/workflows/convert-to-json.yml

```diff
@@ -0,0 +1,41 @@
+name: Convert to JSON
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
I think we should remove the pull request build, and only build on master pushes.